### ISSUE TITLE
Nginx Reverse Proxy - Redirect Loop - Build 5.1-505

### ISSUE THREAD

**Issue by EnderGamingFilms** [plan-player-analytics/Plan#1416]
### Describe the issue
I am running a reverse proxy using Nginx and I was having issues with `Error 402` and realized I was on an older version so I updated to `Plan v5.1-dev-build-505`  and I was suddenly getting `too many redirects`. So after reading #1393 I downloaded `Plan v5.1-dev-build-502` and that seemed to fix everything. I just wanted to let you know so you could fix it at some point.

### Additional information
Nginx Config:
```
server {
    if ($host = stats.some-website.com) {
        return 301 https://$host$request_uri;
    } # managed by Certbot


    listen 80;
    server_name stats.my-website.com;
    return 301 https://stats.my-website.com;
}

server {
    listen 443 ssl;
    server_name stats.my-website.com;

    location / {
        proxy_pass https://proxy-server-ip:8817;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_set_header Host $http_host;
        proxy_intercept_errors on;
    }

    ssl_certificate /etc/letsencrypt/live/stats.my-website.com-0001/fullchain.pem; # managed by Certbot
    ssl_certificate_key /etc/letsencrypt/live/stats.my-website.com-0001/privkey.pem; # managed by Certbot
}
```

Plan Config:
```
Webserver:
    Port: 8817
    Alternative_IP: true
        # %port% is replaced automatically with Webserver.Port
        Address: stats.solarismc.com:
    # InternalIP usually does not need to be changed, only change it if you know what you're doing!
    # 0.0.0.0 allocates Internal (local) IP automatically for the WebServer.
    Internal_IP: 0.0.0.0
    Security:
        SSL_certificate:
            KeyStore_path: PlanCert.jks
            Key_pass: MnD3bU5HpmPXag0e
            Store_pass: wDwwf663NLTm73gL
            Alias: DefaultPlanCert
        # HTTPS is required for Login.
        Disable_authentication: false
        # Cross-Origin Resource Sharing (Requests from non-Plan web pages)
        # https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
        CORS:
            Allow_origin: "*"
    Disable_Webserver: false
    External_Webserver_address: "https://stats.my-website.com"
```

**Comment by EnderGamingFilms** [comment]
Actually just now for some reason, the redirect issue have returned. Idk what to do anymore.

**Comment by Rsl1122** [comment]
This should be resolved with 5.1 build 545 with the replacement of the login mechanism. Let me know if it keeps happening.
